modules/security-context-constraints-example.adoc

```diff
@@ -91,10 +91,15 @@ kind: Pod
 metadata:
   name: security-context-demo
 spec:
+  # TODO: How to update this, with the <1> callout below
```

Comment:
@s-urbaniak I went to move this under spec, but this already has one, plus it has a specific description in the <1> callout below. Not sure what to do with this example. Thoughts?

Reply:
let's leave the examples in this module in their original form

Reply:
Backed out changes to this file